modules/examples/src/main/scala/dev/profunktor/redis4cats/PubSubDemo.scala

```diff
@@ -44,7 +44,7 @@ object PubSubDemo extends LoggerIOApp {
       pub1 = pubSub.publish(eventsChannel)
       pub2 = pubSub.publish(gamesChannel)
     } yield Stream(
-      sub1.through(sink("#events")),
+      sub1.take(1).through(sink("#events")),
```

Comment:
Not sure regarding leaving this in the demo. Can we have a test for dispatcher leakage instead?